terminal/linux_macos-terminal/README.md

```diff
@@ -7,19 +7,68 @@ The challenge with teaching anyone how to use the terminal is the rather large l
 
 A couple must-know commands:
-**pwd**: "Print Working Directory". This command prints your current position within your filesystem.
-Example: `$pwd` -> `/Users/kushdesai/github/ultimate-hackathon-starting-guide/getting-started-terminal/linux_macos-terminal`
-**cd**: Stands for "change directory". Used to navigate into folders
-Example: `cd ../windows-terminal` -> no output, but I am not in the windows-terminal folder. `..` refers to moving back ("up") one directory
-**ls**: "list". This command lists all the files present in the current folder.
-Example: `ls` -> `README.md`, since this README is the only folder in the linux_macos-terminal folder. 
-**touch**: Creates a file
-Example: `touch thisisatest.txt` -> creates a file named thisisatest.txt in the current directory
-**mkdir**: Makes a directory (folder)
-Example: `mkdir this_is_a_folder` -> creates a folder named this_is_a_folder in the current directory. 
-**rm**: "remove". This command is used to remove files and folders from the terminal. NOTE: Use this command with caution, since deleted files cannot be recovered easily. 
-Example 1: `rm thisisatest.txt` -> this removes the text file you created with `touch` in the last example
-Example 2: `rm -r this_is_a_folder` -> the `-r` flag means "recursive", and is necessary to delete folders. This command deletes the folder you created in a previous example. 
+- **pwd**: "Print Working Directory". This command prints your current position within your filesystem.
+    -  Example: `$pwd` -> `/Users/kushdesai/github/ultimate-hackathon-starting-guide/getting-started-terminal/linux_macos-terminal`
+- **cd**: Stands for "change directory". Used to navigate into folders
+    - Example: `cd ../windows-terminal` -> no output, but I am not in the windows-terminal folder. `..` refers to moving back ("up") one directory
+- **ls**: "list". This command lists all the files present in the current folder.
+    - Example: `ls` -> `README.md`, since this README is the only folder in the linux_macos-terminal folder. 
+- **touch**: Creates a file
+    - Example: `touch thisisatest.txt` -> creates a file named thisisatest.txt in the current directory
+- **mkdir**: Makes a directory (folder)
+    - Example: `mkdir this_is_a_folder` -> creates a folder named this_is_a_folder in the current directory. 
+- **rm**: "remove". This command is used to remove files and folders from the terminal. NOTE: Use this command with caution, since deleted files cannot be recovered easily. 
+    - Example 1: `rm thisisatest.txt` -> this removes the text file you created with `touch` in the last example
+    - Example 2: `rm -r this_is_a_folder` -> the `-r` flag means "recursive", and is necessary to delete folders. This command deletes the folder you created in a previous example. 
+
+# Using Makefiles
```

Comment:
lets save this section to include somewhere else - people don't use makefiles just when interacting with the terminal. we could potentially create a section focusing on different programming languages and you could include this in a c/cpp guide

DELETE makefile section